src/js/node/http.ts

```diff
@@ -1411,10 +1497,12 @@ class ClientRequest extends OutgoingMessage {
     this.#bodyChunks.push(...chunks);
     callback();
   }
+
   _destroy(err, callback) {
     this.destroyed = true;
     // If request is destroyed we abort the current response
     this[kAbortController]?.abort?.();
+    this.socket.destroy();
```

Comment:
Is there always going to be a socket here? And should it be destroyed on every call?

Reply:
FakeSocket yes